egs/tunisian_msa/s5/run.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash 
+
+# Trains on 11 hours of speechfrom CTELL{ONE,TWO,THREE,FOUR,FIVE}
+# Uses the QCRI vowelized Arabic lexicon.
+# Converts the Buckwalter encoding to utf8.
+# Uses the perl module Encode::Arabic::Buckwalter for the conversion.
+# On Mac OSX use cpanm (cpanminus) to install perl module.
+. ./cmd.sh
+. ./path.sh
+stage=0
+
+. ./utils/parse_options.sh
+
+set -e
+set -o pipefail
+set u
+
+# Do not change tmpdir, other scripts under local depend on it
+tmpdir=data/local/tmp
+
+# The speech corpus is on openslr.org
+speech="http://www.openslr.org/resources/46/Tunisian_MSA.tar.gz"
+
+# We use the QCRI lexicon.
+lex="http://alt.qcri.org/resources/speech/dictionary/ar-ar_lexicon_2014-03-17.txt.bz2"
+
+# We train the lm on subtitles.
+subs_src="http://opus.nlpl.eu/download.php?f=OpenSubtitles2018/mono/OpenSubtitles2018.ar.gz"
+
+if [ $stage -le 1 ]; then
+  # Downloads archive to this script's directory
+  local/tamsa_download.sh $speech
+
+  local/qcri_lexicon_download.sh $lex
+
+  local/subs_download.sh $subs_src
+fi
+
+# preparation stages will store files under data/
+# Delete the entire data directory when restarting.
+if [ $stage -le 2 ]; then
+  local/prepare_data.sh
+fi
+
+if [ $stage -le 3 ]; then
+  mkdir -p $tmpdir/dict
+  local/qcri_buckwalter2utf8.pl > $tmpdir/dict/qcri_utf8.txt
+fi
+
+if [ $stage -le 4 ]; then
+  local/prepare_dict.sh $tmpdir/dict/qcri_utf8.txt
+fi
+
+if [ $stage -le 5 ]; then
+  # prepare the lang directory
+  utils/prepare_lang.sh data/local/dict "<UNK>" data/local/lang data/lang
+fi
+
+if [ $stage -le 6 ]; then
+  echo "Preparing the subs data for lm training."
+  local/subs_prepare_data.pl 
+fi
+
+if [ $stage -le 7 ]; then
+  echo "lm training."
+  local/prepare_lm.sh  $tmpdir/subs/lm/in_vocabulary.txt
+fi
+
+if [ $stage -le 8 ]; then
+  echo "Making grammar fst."
+  utils/format_lm.sh \
+    data/lang data/local/lm/trigram.arpa.gz data/local/dict/lexicon.txt \
+    data/lang_test
+fi
+
+if [ $stage -le 9 ]; then
+  # extract acoustic features
+  for fld in devtest train test; do
+    steps/make_plp_pitch.sh data/$fld exp/make_plp_pitch/$fld plp_pitch
```

Comment:
I was just wondering, did PLP+pitch work better than MFCCs?

Reply:
I am using the Tunisian MSA corpus in recipes derived from the babel_multilang recipes which use plp_pitch.
Should I use mfcc instead?

Reply:
I see, thanks! I was just wondering, since f0 (pitch) is more commonly used when working with tonal languages, which makes a lot of sense in the multilang setup.